test/postgresql.discover.test.js

```diff
@@ -339,4 +339,14 @@ describe('Discover and build models', function() {
       });
     });
   });
+
+  it('should build a model from discovery with comments', function(done) {
+    db.discoverAndBuildModels('TestGeo', {schema: 'public'}, function(err, schema) {
+      var prop = schema.Testgeo.definition.properties.loc.postgresql;
+      assert(prop);
+      assert.equal(prop.description, 'The location of property of TestGeo');
```

Comment:
It would be good if you can assert [the comment for table](https://github.com/strongloop/loopback-connector-postgresql/pull/341/files#diff-1d86d0e19a7dc8c0ecc368e25741e1a0R59) as well.